src/starknet/starknet_storage.rs

```diff
@@ -107,15 +114,20 @@ impl CommitmentInfo {
     where
         S: Storage + 'static,
         H: HashFunctionType + Sync + Send + 'static,
-        LF: LeafFact<S, H> + Send + 'static,
+        LF: LeafFact<S, H> + Send + 'static + std::fmt::Display,
     {
         let previous_tree_root = Felt252::from_bytes_be_slice(&previous_tree.root);
 
+        for modification in modifications.iter() {
+            println!("{}: {}", modification.0, modification.1);
+        }
```

Comment:
If you want to keep this around, I'd prefix it with some context (e.g. the fn you're in)